metapool/metapool.py

```diff
@@ -1543,9 +1548,13 @@ def compress_plates(compression_layout, sample_accession_df,
             compression_layout[plate_dict_index]['Project Name']
         plate_map['Plate Position'] = position
         plate_map['Project Plate'] = \
-            compression_layout[plate_dict_index]['Project Plate']
+            str(compression_layout[plate_dict_index]['Project Name'] +
+                '_' +
+                compression_layout[plate_dict_index]['Project Plate'])
```

Comment:
If this is PR is not super urgent, then I would advise decomposing this into something like:

```python
def make_project_plate_name(df):
    return f"{df[PROJECT_NAME]}_{df[PROJECT_PLATE]}"
```

...and generally, it would be more robust to refactor and use string literals (e.g., centralize `PROJECT_NAME = 'Project Name'`) to avoid typos and what can be difficult bugs.

Decomposing this makes it easy to test, and easy to change in the future should other information need to be added

Reply:
I understand what you mean. I think we are on the urgent timeline though. We can clean this up as regular maintenance, because I'm sure we will stumble into small issues as we move this absolute quant features into production.

Reply:
I will make an issue out of it in GitHub issues as something we can revisit.